src/interfaces/coral_web/src/components/Shared/Markdown/Markdown.tsx

```diff
@@ -109,8 +113,8 @@ export const Markdown = ({
         components={{
           pre: Pre,
           p: P,
-          // @ts-ignore
           references: References,
+          // @ts-ignore
           iframe: Iframe,
```

Comment:
the `data-src` prop in our custom `Iframe` component isn't compatible with `ReactMarkdown`'s custom component types

Reply:
Ty for explanation